src/RecyclableMemoryStream.cs

```diff
@@ -1530,7 +1540,7 @@ private void EnsureCapacity(long newCapacity)
                 this.memoryManager.ReportStreamOverCapacity(this.id, this.tag, newCapacity, this.AllocationStack);
                 throw new OutOfMemoryException(
                     "Requested capacity is too large: " + newCapacity.ToString(CultureInfo.InvariantCulture) +
-                    ". Limit is " + this.memoryManager.MaximumStreamCapacity.ToString(CultureInfo.InvariantCulture));
+                    ". Limit is " + this.memoryManager.MaximumStreamCapacity.ToString(CultureInfo.InvariantCulture) + ".");
```

Comment:
Let's make this one an interpolated string too.

Reply:
And calling the `ToString()` override is also unnecessary.

Reply:
`ToString` will avoid boxing of the value type.

And to use `CultureInfo.InvariantCulture`, you'd need to use `FormattableString.Invariant($"...")`.

Reply:
@paulomorgado  Didn't know about the `FormattableString.Invariant($"...")`, thanks!
But does it seem overcomplicated to think about allocating memory in a location that throws the OutOfMemoryException? It feels like a very rare case that should not be necessary optimized.

Reply:
> @paulomorgado Paulo Morgado FTE Didn't know about the `FormattableString.Invariant($"...")`, thanks!
> But does it seem overcomplicated to think about allocating memory in a location that throws the OutOfMemoryException? It feels like a very rare case that should not be necessary optimized.
> 
> And as I see [here](https://sharplab.io/#v2:D4AQTAjAsAUCAMACEEB0BxANgewEYENMBLAL3wBcjsA7AbllhAGZkxEBhRAb1kT+RYgALIgCyACgCU3Xvzk5qAc0TUApgHd2+AA74AxkXIBPRAF5E8ejDlzyACwBO2dSo2IA8gFdy7gGajVAFtsByMAUQAPPVVtShpxADEQwIpyfFxMVQBlcgciJVQASWoAN3w8/GpycQASACIAJVUAR09VAGdyVQATRD0dfUMTInbEcmxsRExyxVUALm41TQGDYwBfVDrJSSs5Ndg1oA===), `FormattableString.Invariant($"...")` will not protect us from boxing anyway.

I was not implying that it was better, or even good. I was just saying that it was the only way of getting the same outcome out of an interpolated string.

Just use an interpolated string with the `.ToString(CultureInfo.InvariantCulture)` invocation in the holes. If all holes are to be filled with strings, the compiler will choose `string.Concat(string[])` instead of `string.Format`.

Reply:
Got it, done.